IBF-Typhoon-model/.gitignore

```diff
@@ -1,15 +1,10 @@
 # Cutsom by Aki
 forecast/
-typhoon_infographic/
```

Comment:
To avoid confusion, there should only be one `.gitignore` file at the top level of the project. If you want to ignroe some of these items, then you should add them to the top-level file.